addon/helpers/next.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+import getIndex from '../utils/get-index';
+import createNeedleHaystackHelper from '../-private/create-needle-haystack-helper';
+
+let { isPresent } = Ember;
+
+export function next(array, currentValue) {
+  let currentIndex = getIndex(array, currentValue);
+  return isPresent(currentIndex) ? array[currentIndex + 1] : null;
```

Comment:
In the event that the `currentValue` is the last item in the array, I think we would want to return the item instead of `undefined`